### PR TITLE
Add Callouts preview accordion on SubspacesList default subspace template selector

### PR DESCRIPTION
**Pull request by ccanos**
#6879

- moved the Callouts accordion from the CollaborationTemplatePreview to its own file as a component.
- Added that new component to SubspacesList

## Summary by CodeRabbit

- **New Features**
	- Introduced the `InnovationFlowCalloutsPreview` component for displaying callouts in an expandable format.
	- Enhanced the `AdminChallengesPage` GraphQL query to include detailed callout information.
	- Integrated the `InnovationFlowCalloutsPreview` component into the `SubspaceListView` for improved callout visualization.

- **Bug Fixes**
	- Updated state management in `SubspaceListView` to handle selected state toggling correctly.

- **Refactor**
	- Simplified the `CollaborationTemplatePreview` component by replacing the accordion structure with the new callouts preview component, improving maintainability.

### CODE REVIEW

File: src/domain/collaboration/callout/CalloutsPreview/InnovationFlowCalloutsPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+const InnovationFlowCalloutsPreview = ({ callouts, selectedState, loading }: InnovationFlowCalloutsPreviewProps) => {
+  const [selectedCallout, setSelectedCallout] = useState<string | false>(false);
+  const handleSelectedCalloutChange = (calloutId: string) => (_event, isExpanded: boolean) =>
+    setSelectedCallout(isExpanded ? calloutId : false);
+
+  return (
+    <>
+      {!loading && callouts && (
+        <Box>
+          {callouts
```
Comment (hero101) [inline src/domain/collaboration/callout/CalloutsPreview/InnovationFlowCalloutsPreview.tsx:81]: very inefficient - can you memo?

Comment (hero101) [reply src/domain/collaboration/callout/CalloutsPreview/InnovationFlowCalloutsPreview.tsx:81]: I might be wrong but it seems like you can memo the whole rendered component.

File: src/domain/collaboration/callout/CalloutsPreview/InnovationFlowCalloutsPreview.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  border: `1px solid ${theme.palette.divider}`,
+  borderRadius: theme.shape.borderRadius,
+  marginBottom: gutters(1)(theme),
+  boxShadow: 'none',
+}));
+
+const CalloutDescription = ({ callout }: { callout: CalloutPreview }) => {
+  const { t } = useTranslation();
+
+  switch (callout.type) {
```
Comment (hero101) [inline src/domain/collaboration/callout/CalloutsPreview/InnovationFlowCalloutsPreview.tsx:51]: can this be memo`d?

**Review by hero101 — CHANGES_REQUESTED** [review]

**Review by valentinyanakiev — DISMISSED** [review]

File: src/domain/collaboration/callout/CalloutsPreview/InnovationFlowCalloutsPreview.tsx (hunk 2 above)
Comment (hero101) [reply src/domain/collaboration/callout/CalloutsPreview/InnovationFlowCalloutsPreview.tsx:51]: bump

**Review by hero101 — CHANGES_REQUESTED** [review]
the rabbit has a good suggestion, we might have up to 10-15 items, which are not that many, but still it's up to you.

**Review by hero101 — APPROVED** [review]

**Pull request merged**